wsgi/odyssy/events/views.py

```diff
@@ -0,0 +1,24 @@
+from django.shortcuts import get_object_or_404
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.shortcuts import render
+from .models import Event
+
+
+def event(request):
+    events_list = Event.objects.order_by('-event_date')
+    paginator = Paginator(events_list, 3)
+    page = request.GET.get('page')
```

Comment:
Default value is set as 1. If `request.GET.get('page')` returns `None`, then it wil raise `PageNotAnInteger` Exception. This exception is catched and appropriate action is performed here.